guidelines/MONAI-Workloads.md

```diff
@@ -0,0 +1,1149 @@
+
+
+### Research and Development {#research-and-development}
+
+
+#### Retrospective Study {#retrospective-study}
+
+As part of AI model development, single AI algorithms are evaluated against existing studies stored in the PACS. In other cases, new datasets are made available and researchers look to gain new insights or data from an AI model to form a prospective diagnosis.  These studies will have on average greater than 10K units to be processed to satisfy statistical variety constraints and pool diversity requirements.  
+
+**Clinical Workflow**
+
+
+
+1. A researcher or model developer orders a selective batch of radiology exams to be processed with a specific AI model. 
+1. Clinicians and researchers working on the algorithm AI model defines a cohort selection criteria and gathers a selection of patient and study IDs.
+2. The DICOM router or some application with access to the PACs system has the ability to fetch data based on the selection criteria.  The application (either on DICOM router or externally) has the ability to send data to multiple devices efficiently.  In addition, the application has the ability to load balance and throttle the number of studies sent to a particular destination.    
+3. DICOM router application requests the original DICOM study image from the PACS system at a rate of 8 studies per minute.  DICOM router and AI processing engine are connected to each other with a high speed 10gbps network connection.  
+4. DICOM study gets processed by the AI processing engine, including pre-processing, inference and post processing.  The output of AI processing will return as an example will return a breast density score.  The result will be packaged into a report or optionally written on to a modified DICOM image.  The size of the resulting object will be the size of a DICOM image.  
+5. AI processing engine sends the study further to DICOM routing.
+6. DICOM router sends the study further to PACS, RIS or some external study result database.  
+7. The researcher responsible for ordering the study or reviewing the results will review the AI platform success and failure rate for processing the images.  
+
+**Clinical Use Case:** Group AI algorithm study
+
+**Computational Workload:** [Batch](#batch-computational-workload)
+
+**Clinical workload:** 1500 (check histograms)  studies per day  for greater than 7 days with no manual intervention or study processing failures. study size N x M x K depending on modality and requested study type. See Table for estimates of study size to modality and study type mapping.
+
+**Importance to Clinical AI Deployment:** medium, the use case will drive up the quality and model generalization but will not directly impact AI use as part of the radiology workflow.
+
+**Figure:** Retrospective Study
+
+![MONAI_Retrospective Study](assets/MONAI_Retrospective_Study.jpg)
+
+
+### Clinical Workloads Summary {#clinical-workloads-summary}
+
+
+<table>
```

Comment:
oh this bad -- please don't do tables like this. Github Markdown (GHMD) has built-in table support. The number one rule when writing markdown is **do not use markup**.